### PR TITLE
[TRUS-3095] - Residency question(s) for Charity Beneficiary in Trust Registration

### PR DESCRIPTION
**Pull request by jassalrichy**

### CODE REVIEW

File: app/controllers/register/beneficiaries/charityortrust/charity/AddressInTheUkYesNoController.scala [hunk 1]
```diff
@@ -18,7 +18,7 @@ package controllers.register.beneficiaries.charityortrust.charity
 
 import config.annotations.CharityBeneficiary
 import controllers.actions._
-import controllers.actions.register.company.NameRequiredAction
+import controllers.actions.register.trust.NameRequiredAction
```
Comment (BriWak) [inline app/controllers/register/beneficiaries/charityortrust/charity/AddressInTheUkYesNoController.scala:21]: should be import controllers.actions.register.**charity**.NameRequiredAction

File: app/controllers/register/beneficiaries/charityortrust/charity/AddressYesNoController.scala [hunk 2]
```diff
@@ -18,7 +18,7 @@ package controllers.register.beneficiaries.charityortrust.charity
 
 import config.annotations.CharityBeneficiary
 import controllers.actions._
-import controllers.actions.register.company.NameRequiredAction
+import controllers.actions.register.trust.NameRequiredAction
```
Comment (BriWak) [inline app/controllers/register/beneficiaries/charityortrust/charity/AddressYesNoController.scala:21]: charity again

File: app/controllers/register/beneficiaries/charityortrust/charity/AmountDiscretionYesNoController.scala [hunk 3]
```diff
@@ -18,7 +18,7 @@ package controllers.register.beneficiaries.charityortrust.charity
 
 import config.annotations.CharityBeneficiary
 import controllers.actions.StandardActionSets
-import controllers.actions.register.company.NameRequiredAction
+import controllers.actions.register.trust.NameRequiredAction
```
Comment (BriWak) [inline app/controllers/register/beneficiaries/charityortrust/charity/AmountDiscretionYesNoController.scala:21]: charity again

File: app/controllers/register/beneficiaries/charityortrust/charity/CharityAddressUKController.scala [hunk 4]
```diff
@@ -18,7 +18,7 @@ package controllers.register.beneficiaries.charityortrust.charity
 
 import config.annotations.CharityBeneficiary
 import controllers.actions._
-import controllers.actions.register.company.NameRequiredAction
+import controllers.actions.register.trust.NameRequiredAction
```
Comment (BriWak) [inline app/controllers/register/beneficiaries/charityortrust/charity/CharityAddressUKController.scala:21]: charity again

File: app/controllers/register/beneficiaries/charityortrust/charity/CharityAnswersController.scala [hunk 5]
```diff
@@ -17,7 +17,7 @@
 package controllers.register.beneficiaries.charityortrust.charity
 
 import controllers.actions._
-import controllers.actions.register.company.NameRequiredAction
+import controllers.actions.register.trust.NameRequiredAction
```
Comment (BriWak) [inline app/controllers/register/beneficiaries/charityortrust/charity/CharityAnswersController.scala:20]: charity again

File: app/controllers/register/beneficiaries/charityortrust/charity/CharityInternationalAddressController.scala [hunk 6]
```diff
@@ -18,7 +18,7 @@ package controllers.register.beneficiaries.charityortrust.charity
 
 import config.annotations.CharityBeneficiary
 import controllers.actions._
-import controllers.actions.register.company.NameRequiredAction
+import controllers.actions.register.trust.NameRequiredAction
```
Comment (BriWak) [inline app/controllers/register/beneficiaries/charityortrust/charity/CharityInternationalAddressController.scala:21]: charity again

File: app/controllers/register/beneficiaries/charityortrust/charity/HowMuchIncomeController.scala [hunk 7]
```diff
@@ -18,7 +18,7 @@ package controllers.register.beneficiaries.charityortrust.charity
 
 import config.annotations.CharityBeneficiary
 import controllers.actions.StandardActionSets
-import controllers.actions.register.company.NameRequiredAction
+import controllers.actions.register.trust.NameRequiredAction
```
Comment (BriWak) [inline app/controllers/register/beneficiaries/charityortrust/charity/HowMuchIncomeController.scala:21]: charity again

**Review by BriWak — CHANGES_REQUESTED** [review]

File: app/controllers/register/beneficiaries/charityortrust/charity/AddressInTheUkYesNoController.scala (hunk 1 above)
Comment (jassalrichy) [reply app/controllers/register/beneficiaries/charityortrust/charity/AddressInTheUkYesNoController.scala]: Updated

File: app/controllers/register/beneficiaries/charityortrust/charity/AddressYesNoController.scala (hunk 2 above)
Comment (jassalrichy) [reply app/controllers/register/beneficiaries/charityortrust/charity/AddressYesNoController.scala]: Updated

File: app/controllers/register/beneficiaries/charityortrust/charity/AmountDiscretionYesNoController.scala (hunk 3 above)
Comment (jassalrichy) [reply app/controllers/register/beneficiaries/charityortrust/charity/AmountDiscretionYesNoController.scala]: Updated

File: app/controllers/register/beneficiaries/charityortrust/charity/CharityAddressUKController.scala (hunk 4 above)
Comment (jassalrichy) [reply app/controllers/register/beneficiaries/charityortrust/charity/CharityAddressUKController.scala]: Updated

File: app/controllers/register/beneficiaries/charityortrust/charity/CharityAnswersController.scala (hunk 5 above)
Comment (jassalrichy) [reply app/controllers/register/beneficiaries/charityortrust/charity/CharityAnswersController.scala]: Updated

File: app/controllers/register/beneficiaries/charityortrust/charity/CharityInternationalAddressController.scala (hunk 6 above)
Comment (jassalrichy) [reply app/controllers/register/beneficiaries/charityortrust/charity/CharityInternationalAddressController.scala]: Updated

File: app/controllers/register/beneficiaries/charityortrust/charity/HowMuchIncomeController.scala (hunk 7 above)
Comment (jassalrichy) [reply app/controllers/register/beneficiaries/charityortrust/charity/HowMuchIncomeController.scala]: Updated

**Review by BriWak — APPROVED** [review]

**Pull request merged**